algebras/algebra/src/test/scala/endpoints/algebra/client/EndpointsTestSuite.scala

```diff
@@ -281,10 +281,6 @@ trait EndpointsTestSuite[T <: EndpointsTestApi] extends ClientTestBase[T] {
           ) shouldEqual "/foo?id=f4b9defa-1ad8-453f-9a06-2683b8564b8d"
         }
 
-        "escaping" in {
-          encodeUrl(path /? qs[String]("q"))("foo bar/baz") shouldEqual "?q=foo+bar%2Fbaz"
```

Comment:
Would it be possible to keep this test but change the assertion to accept both `"?q=foo+bar%2Fbaz"` and `?q=foo%20bar%2Fbaz` as discussed in https://github.com/julienrf/endpoints/pull/499#issuecomment-602168809 ?

Reply:
Oh yeah, sorry about that.